Rakefile

```diff
@@ -1,15 +1,17 @@
 require 'rubygems'
 require 'bundler'
+require_relative 'ext/ext_helper'
 
-Bundler::GemHelper.install_tasks
 
 def ext_dependencies(name)
-  FileList["ext/#{name}/**/*"].reject { |file| file =~ /\b(Release|Debug)\b/ }
+  FileList["ext/#{name}/**/*"].reject { |file| file =~ /\b(Release|Debug|x86Release|x86Debug|x64Release|x64Debug)\b/ }
 end
 
 def ms_build(name)
   name = File.basename(name, File.extname(name))
-  sh "msbuild /property:Configuration=Release ext\\#{name}\\#{name}.sln"
+  cmd = "msbuild /p:Configuration=Release ext\\#{name}\\#{name}.sln"
+  cmd += " && #{cmd} /p:Platform=x64" unless name == 'WindowsForms'
```

Comment:
I'm not too familiar with `msbuild` to be honest - do I undertand correctly if `Platform=x64` is not specified then `msbuild` always builds `x86` version only?

Reply:
Correct. When no platform is specified it will compile for x86